src/app/shell/admin-tools/platform/directions/directions.component.ts

```diff
@@ -80,6 +80,7 @@ export class DirectionsComponent implements OnInit, OnDestroy {
 
   onItemsPerPageChange(itemsPerPage: number): void {
     this.store.dispatch(new SetDirectionsPerPage(itemsPerPage));
+    this.store.dispatch(new GetFilteredDirections());
```

Comment:
this.store.dispatch([ew SetDirectionsPerPage(itemsPerPage), new GetFilteredDirections()]);